### ISSUE TITLE
How can I use this repo?

### ISSUE THREAD

**Issue by HaoKunT** [skswlsaks/FastMCDArma#1]
Is this repo finished? Or can I invoke the `Execution::step4()` to calculate the MCD？

**Comment by skswlsaks** [comment]
Yes, you can use the repo. Since I haven't managed this repo, im not sure this is working or not! Please use it at your own risk.

**Comment by skswlsaks** [comment]
Thanks for your interest in my code!

**Comment by HaoKunT** [comment]
@skswlsaks so the `fastMCD` is finished? There are many comments in source code. So I don't known whether it have been finished or not.
